types/shared/Responses.ts

```diff
@@ -1,4 +1,4 @@
-import {URL} from '../../shared/utils';
+import {URL} from './utils';
 
 /**
  * @id #Responses
```

Comment:
Note that `Responses` are now shared between the `application` & `prototypeApplication`, but nearly all else is re-defined fresh.

The `shared` folder is generally still sparsely populated, this will make more sense to refactor once `prototypeApplication` _replaces_ `application` in my opinion ! It's felt much easier to just keep them totally seperate while doing this comparison/migration exercise rather than try to refactor/share more in same go.